robots.txt

```diff
@@ -0,0 +1,21 @@
+User-agent: *
+Disallow: /blog/create
+Crawl-delay: 8
```

Comment:
Why a delay here?

Reply:
"Crawl-delay: The crawl delay command is meant to stop search engine spider bots from overtaxing a server. It allows administrators to specify how long the bot should wait between each request, in milliseconds."

Thought it doesn't hurt but we also don't really need it. I can remove it if you want.

Reply:
Please remove it :)